### PR TITLE
Update device-onboarding-macos-overview.md

### PR DESCRIPTION
**Pull request by arunt14**
Adding conditions supported on Mac, @chrfox for your review

### CODE REVIEW

File: microsoft-365/compliance/device-onboarding-macos-overview.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ Endpoint DLP supports these browsers on macOS (three latest released versions):
 
 See, [Microsoft 365 licensing guidance for information protection](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#information-protection-data-loss-prevention-for-exchange-online-sharepoint-online-and-onedrive-for-business).
 
+## Conditions supported on macOS 
+
+Once a macOS device is onboarded into Microsoft Purview solutions, you can use the following condition with data loss prevention (DLP) policies
+
+**Content Contains** – Apply to documents that contain sensitive information types, sensitivity labels
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/device-onboarding-macos-overview.md]:
```suggestion
Once a macOS device is onboarded into Microsoft Purview solutions, you can use the following condition with data loss prevention (DLP) policies:

**Content Contains** – Applies to documents that contain sensitive information types and sensitivity labels. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arunt14, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
@arunt14  do you intend to say that macOS only supports the content contains condition? Can you please review https://learn.microsoft.com/en-us/microsoft-365/compliance/dlp-policy-reference?view=o365-worldwide#conditions-devices-supports for which conditions are supported by Windows 10/11 and which are supported by macOS?  The DLP policy reference is where this is better documented

**Review by arunt14 — COMMENTED** [review]
This looks good to be published @chrfox

**Pull request merged**